src/coordsim/network/flow.py

```diff
@@ -10,7 +10,7 @@
 class Flow:
 
     def __init__(self, flow_id, sfc, dr, size, creation_time, destination=None, egress_node_id=None, current_sf=None,
-                 current_node_id=None, current_position=0, end2end_delay=0.0):
+                 current_node_id=None, current_position=0, end2end_delay=0.0, ttl=50):

```

Comment:
It's not a good idea to provide an arbitrary default TTL. If someone isn't aware of this and doesn't set the TTL, I think the expectation would be that there is no TTL. Would lead to unexpected and hard-to-debug outcomes.

I'd set `ttl=None` by default. Do we still support flows without TTL? Then that should mean no TTL.
Otherwise, you can raise an exception saying that the TTL needs to be specified.

Reply:
We do not support flows with no TTL anymore. I will raise an exception then when no TTL is specified in the config.